example/src/SiteWrapper.react.js

```diff
@@ -16,6 +16,10 @@ type Props = {|
   +children: React.Node,
 |};
 
+type State = {|
+  +unreadCount: number,
```

Comment:
We never display the number of unread messages, so we could also just make this a `hasUnread: boolean,` but I'm good with whatever you think is better here.

Reply:
To be honest either is fine with me. Using unreadCount gives us a chance to show the number of notifications as a badge instead of just the dot, but if thats never going to be in the roadmap then a bool will be fine. Either way the implementor will need to do some kind of reduce an the array of notifications to get the count or simply the bool.

My only other thought would be to provide a flag on the notification itself if its read or not - so it can be styled accordingly - and the unread badge could be shown if there are any unread notifications. That way the developer only needs to mark the notification as read and everything else could just work. implementation of the 'mark all read' would be a case of running the callback on each of the messages. But that would be for another pull reaquest. If you like that idea I could look at the implementation in the near future as our project evolves I will be needing to do something like this anyway - question is do you want it abstracted into your project or left as an implementation detail for users?